test/markup/dts/default.expect.txt

```diff
@@ -10,62 +10,62 @@
-			interrupts = <span class="hljs-params">&lt;<span class="hljs-number">0</span> <span class="hljs-number">25</span> <span class="hljs-number">4</span>&gt;</span>;
-			reg = <span class="hljs-params">&lt;<span class="hljs-number">0xe0004000</span> <span class="hljs-number">0x1000</span>&gt;</span>;
+		i2c0:</span> <span class="hljs-title class_">i2c@e0004000</span> <span class="hljs-punctuation">{</span>
+			<span class="hljs-attr">compatible</span> <span class="hljs-operator">=</span> <span class="hljs-string">&quot;cdns,i2c-r1p10&quot;</span><span class="hljs-punctuation">;</span>
+			<span class="hljs-attr">status</span> <span class="hljs-operator">=</span> <span class="hljs-string">&quot;disabled&quot;</span><span class="hljs-punctuation">;</span>
+			<span class="hljs-attr">clocks</span> <span class="hljs-operator">=</span> <span class="hljs-params">&lt;<span class="hljs-variable">&amp;clkc</span> <span class="hljs-number">38</span>&gt;</span><span class="hljs-punctuation">;</span>
+			<span class="hljs-attr">interrupt-parent</span> <span class="hljs-operator">=</span> <span class="hljs-params">&lt;<span class="hljs-variable">&amp;intc</span>&gt;</span><span class="hljs-punctuation">;</span>
+			<span class="hljs-attr">interrupts</span> <span class="hljs-operator">=</span> <span class="hljs-params">&lt;<span class="hljs-number">0</span> <span class="hljs-number">25</span> <span class="hljs-number">4</span>&gt;</span><span class="hljs-punctuation">;</span>
+			<span class="hljs-attr">reg</span> <span class="hljs-operator">=</span> <span class="hljs-params">&lt;<span class="hljs-number">0xe0004000</span> <span class="hljs-number">0x1000</span>&gt;</span><span class="hljs-punctuation">;</span>
 			<span class="hljs-meta">#address-cells = &lt;1&gt;;</span>
 			<span class="hljs-meta">#size-cells = &lt;0&gt;;</span>
-		};
+		<span class="hljs-punctuation">};</span>
 <span class="hljs-symbol">
-		L2:</span> cache-<span class="hljs-class">controller@f8f02000 </span>{
-			compatible = <span class="hljs-string">&quot;arm,pl310-cache&quot;</span>;
-			reg = <span class="hljs-params">&lt;<span class="hljs-number">0xF8F02000</span> <span class="hljs-number">0x1000</span>&gt;</span>;
-			interrupts = <span class="hljs-params">&lt;<span class="hljs-number">0</span> <span class="hljs-number">2</span> <span class="hljs-number">4</span>&gt;</span>;
-			arm,data-latency = <span class="hljs-params">&lt;<span class="hljs-number">3</span> <span class="hljs-number">2</span> <span class="hljs-number">2</span>&gt;</span>;
-			arm,tag-latency = <span class="hljs-params">&lt;<span class="hljs-number">2</span> <span class="hljs-number">2</span> <span class="hljs-number">2</span>&gt;</span>;
-			cache-unified;
-			cache-level = <span class="hljs-params">&lt;<span class="hljs-number">2</span>&gt;</span>;
-		};
+		L2:</span> <span class="hljs-title class_">cache-controller@f8f02000</span> <span class="hljs-punctuation">{</span>
```

Comment:
big fan of these changes! :D

Reply:
How so?

Reply:
I like very detailed highlighting, so when I see stuff like `=`, even though a single character, has its own highlight token, that makes me excited, esp for making themes